src/NATS.Client.KeyValueStore/Internal/NatsKVWatcher.cs

```diff
@@ -358,7 +358,7 @@ private async ValueTask<INatsJSConsumer> CreatePushConsumer(string origin)
             DeliverPolicy = ConsumerConfigDeliverPolicy.All,
             AckPolicy = ConsumerConfigAckPolicy.None,
             DeliverSubject = _sub.Subject,
-            FilterSubject = _filter,
+            FilterSubjects = _filters,
```

Comment:
filter-subjects (plural) was introduced in server 2.10. to keep 2.9 compat we should still use filter-subject (singular) if there is only one filter subject. (which should fix the ci failure)

Reply:
https://github.com/rickdotnet/nats.net.v2/blob/7548b7481ff2cb4767cdf577563a83a4438f7237/src/NATS.Client.KeyValueStore/Internal/NatsKVWatcher.cs#L370